addon/components/select-input.js

```diff
@@ -30,6 +32,20 @@ export default AbstractInput.extend({
   // Computed Properties
   // ==========================================================================
 
+  @readOnly
+  @computed('store.formValue')
+  disabled (value) {
+    const modelDef = this.get('model')
+    const cellConfig = this.get('cellConfig')
+    const bunsenId = this.get('bunsenId')
```

Comment:
I'm just very cautious about what my computed properties depend on.  As far as I know, I think I only want this property recomputed (i.e. re-rendered) when the `formValue` changes.  I suppose you can say the same for `model`, `cellConfig`, and `bunsenId`.

Well, since those props aren't modified asynchronously, maybe I'm worrying about nothing.